code/obj/item/device/sacred_heart.dm

```diff
@@ -0,0 +1,61 @@
+#define HEART 1
+
+/obj/item/device/sacred_heart_scroll
+	name = "Sacred Heart Scroll"
+	icon_state = "sacred_heart_scroll-"
+	desc = "A dusty old scroll containing essoteric knowlege from a time of cloak and dagger"
+	w_class = 2
+	is_syndicate = 1
+	var/implant = /obj/item/organ/heart/sacred
+	var/implants_available = HEART
+	var/list/parts_to_remove = list()
+	var/list/parts_to_add = list()
+
+	attack_self(mob/user as mob)
+		if (ishuman(user))
+			var/mob/living/carbon/human/H = user
+			src.add_fingerprint(H)
```

Comment:
unneeded `src`

Reply:
using `src.variable` is preferred I think

Reply:
I'm confused on this, is it fine or not wanted? I tried without a src and it didn't work so I just left it